package.json

```
@@ -44,12 +44,12 @@
     "wary": "^1.1.1"
   },
   "dependencies": {
+    "balena-image-fs": "^7.0.6",
     "balena-semver": "^2.2.0",
     "bluebird": "^3.7.2",
     "lodash": "^4.17.15",
-    "reconfix": "0.1.0",
-    "resin-device-operations": "^1.6.0",
-    "resin-image-fs": "^5.0.9",
+    "reconfix": "^1.0.0-v0-1-0-fork-46760acff4d165f5238bfac5e464256ef1944476",
```

Any reason the fork can't be published as v0.1.1?

---

Good question, we would probably have to merge this https://github.com/balena-io/reconfix/pull/61 I'll let @Ereski weigh in.

---

@Page- @mbalamat the reconfix repo is a mess. We just found out that `master` has no common commits with the `v0.1.0` tag. So a `v0.1.1` tag based on the `v0.1.0` work requires some non-trivial repo surgery.

---

@Page- I guess it's ok to merge this since reconfix will change soon, right?

---

You'll need to make it
```suggestion
    "reconfix": "1.0.0-v0-1-0-fork-46760acff4d165f5238bfac5e464256ef1944476",
```
then, otherwise the moment a real 1.0.0, or possibly just a different pre-release version, is created then the module will potentially switch to using that and break for anyone npm installing from that point forwards

---

ok done!